cogs/highlight.py

```diff
@@ -270,6 +270,9 @@ def _is_registered(self, guild_idx, guild_id, user_id):
             time = msg.timestamp
             time = time.replace(tzinfo=timezone.utc).astimezone(tz=None).strftime('%H:%M:%S %Z')
             embed_msg += "[{0}] {1.author.name}#{1.author.discriminator}: {1.content}\n".format(time,msg)
+            escaped_msg = embed_msg

```

Comment:
You're adding the existing `embed_msg` to itself.  Consider what would happen if embed_msg was `"test "`, and the new message is `"new message "` (before escaping).  After this code block, it would become "test new message test new message" instead of "test new message".

You'll have to test this, but you probably want something like:
```suggestion
            escaped_msg = "[{0}] {1.author.name}#{1.author.discriminator}: {1.content}\n".format(time,msg)
            escaped_msg = escaped_msg.replace("`", "\`")
            embed_msg += escaped_msg
```